.dockerfiles/opam

```diff
@@ -0,0 +1,18 @@
+ARG DISTRO
+FROM ocaml/opam2:${DISTRO}
+
+WORKDIR /home/opam/src
+USER opam
+COPY *.opam .
+RUN opam pin add --no-action duniverse .
+RUN opam update
+RUN opam depext duniverse ocamlformat=0.9
+RUN opam install ocamlformat=0.9
+RUN opam install --deps-only -t duniverse
+COPY . .
+RUN sudo chown -R opam /home/opam/src
```

Comment:
`COPY` has a `--chown` option ([doc](https://docs.docker.com/engine/reference/builder/#copy)) so you can write this as:
```
COPY --chown=opam:opam . .
```

Reply:
Uuuhh, neat!

Reply:
Hmm it seems it's not working for some reason. Maybe it's not setting the ownership recursively :/

Reply:
I'm a little surprised as it works perfectly fine when I'm running both `docker build` on my machine.

Reply:
Also `alpine` seems unhappy about whatever docker translates the `COPY --chown=opam` to so I'll revert those changes, too bad.